crates/cairo-lint-core/src/fix.rs

```diff
@@ -58,9 +58,25 @@ impl Fixer {
             first_expr.expression(db).as_syntax_node().get_text_without_trivia(db),
         )
     }
+
+    pub fn fix_double_parens(&self, db: &dyn SyntaxGroup, node: SyntaxNode) -> String {
+        let mut expr = Expr::from_syntax_node(db, node.clone());
+
+        while let Expr::Parenthesized(inner_expr) = expr {
+            expr = inner_expr.expr(db);
+        }
+
+        format!(
+            "{}{}\n",
```

Comment:
This should fix the formatting pb
```suggestion
            "{}{}",
```